### PR TITLE
test(editor): Add e2e tests for inline expression editor

### PR DESCRIPTION
**Pull request by ivov**
```
 (Run Finished)


       Spec                                              Tests  Passing  Failing  Pending  Skipped
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  0-smoke.cy.ts                            00:03        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  1-workflows.cy.ts                        00:11        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  10-undo-redo.cy.ts                       01:41       18       18        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  11-inline-expression-editor.cy.ts        00:18        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  2-credentials.cy.ts                      00:12        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  3-default-owner.cy.ts                    00:12        6        6        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  4-node-creator.cy.ts                     00:31        7        7        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  5-ndv.cy.ts                              00:13        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  6-code-node.cy.ts                        00:10        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  7-workflow-actions.cy.ts                 00:41       16       16        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  8-http-request-node.cy.ts                00:06        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  9-expression-editor-modal.cy.ts          00:18        4        4        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        04:42       72       72        -        -        -

```

### CODE REVIEW

File: cypress/e2e/11-inline-expression-editor.cy.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		cy.task('reset');
+		cy.skipSetup();
+	});
+
+	beforeEach(() => {
+		WorkflowPage.actions.visit();
+		WorkflowPage.actions.addInitialNodeToCanvas('Manual Trigger');
+		WorkflowPage.actions.addNodeToCanvas('Hacker News');
+		WorkflowPage.actions.openNodeNdv('Hacker News');
+		WorkflowPage.actions.openInlineExpressionEditor();
```
Comment (ivov) [inline cypress/e2e/11-inline-expression-editor.cy.ts:16]: Is there a way to move the content of `beforeEach` to `before` so the tests share setup and so run faster? I tried to but the run fails after the first test, redirecting to `about:blank`. Or is it best practice to keep all tests in a suite running from scratch?

Comment (MiloradFilipovic) [reply cypress/e2e/11-inline-expression-editor.cy.ts:16]: I guess it depends on what you are trying to do, but I guess having a clean state before each test is in line with cypress guidelines of writing independent tests.

**Review by MiloradFilipovic — APPROVED** [review]
LGTM

**Comment by janober** [comment]
Got released with `n8n@0.208.0`

**Pull request merged**